packages/sdk-browser/src/sdk-client.ts

```diff
@@ -160,13 +160,28 @@ export class TurnkeyBrowserSDK {
   logoutUser = async (): Promise<boolean> => {
     await removeStorageValue(StorageKeys.CurrentUser);
     await removeStorageValue(StorageKeys.AuthBundle);
+    await removeStorageValue(StorageKeys.SigningSession);
 
     return true;
   };
 
   getAuthBundle = async (): Promise<string | undefined> => {
     return await getStorageValue(StorageKeys.AuthBundle);
   };
+
+  getSigningSession = async (): Promise<SigningSession | undefined> => {
+    const signingSession: SigningSession | undefined = await getStorageValue(StorageKeys.SigningSession);
+    if (signingSession) {
+      if (signingSession.expirationTimestamp > Date.now()) {
```

Comment:
If we're always going to expect a valid `expirationTimestamp` to be provided, then should we just combine these two lines? --> `signingSession?.expirationTimestamp...`